srtcore/core.cpp

```diff
@@ -174,8 +174,8 @@ void CUDT::construct()
     // Initilize mutex and condition variables
     initSynch();
 
-    // Default: 
-    m_cbPacketArrival.set(this, &CUDT::defaultPacketArrival);
+    // XXX: Unblock, when the callback is implemented
+    // m_cbPacketArrival.set(this, &CUDT::defaultPacketArrival);
```

Comment:
Why this change is here?

Reply:
Because it's not implemented. This is a stub that should be filled with an implementation that processes the packet for the sake of group receiver buffer, if provided. Also the call to this callback isn't currently properly defined.